test/models/inheritanceClass.ts

```diff
@@ -9,6 +9,14 @@ import { arrayProp, getModelForClass, modelOptions, prop } from '../../src/typeg
 export class Building {
   @prop({ default: 100 })
   public width: number;
+
+  public get calculatedWidth() {
+    return this.width;
+  }
+
+  public get assignedGardenArea() {
```

Comment:
why is this extra getter needed?

Reply:
To make sure it's -somehow- not being overridden or being removed, and passed through the schema tree when building the `Skyscraper` class schema.

Reply:
ok, just wanted to know if this was on purpose and not an "forgotten thing"